### PR TITLE
Updates the pipeline docs page

### PR DESCRIPTION
**Pull request by JanuszL**
- removes duplicated pipeline description from the pipeline docs page
- makes Pipeline.current() method documented
- adds an example of how to use Pipeline.current() in the Pipeline Decorator
  section

Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

#### Why we need this PR?
*Pick one, remove the rest*
- It updates the pipeline docs page
- 
#### What happened in this PR?
*Fill relevant points, put NA otherwise. Replace anything inside []*
 - What solution was applied:
      removes duplicated pipeline description from the pipeline docs page
      makes Pipeline.current() method documented
      adds an example of how to use Pipeline.current() in the Pipeline Decorator
  section
 - Affected modules and functionalities:
     pipeline docs page
 - Key points relevant for the review:
     Wording in the example
 - Validation and testing:
     CI
 - Documentation (including examples):
     Pipeline doc page was updated

**JIRA TASK**: *[NA]*

### CODE REVIEW

**Comment by JanuszL** [comment]
!build

File: dali/python/nvidia/dali/pipeline.py [hunk 1]
```diff
@@ -1274,6 +1275,16 @@ def my_pipe():
         They may result in unwanted, silent hijacking of some arguments of the same name by
         Pipeline constructor. Code written this way would cease to work with future versions of DALI
         when new parameters are added to the Pipeline constructor.
+
+    To obtain a value of the argument added and consumed by the ``@pipeline_def`` the user need to
+    to use :meth:`nvidia.dali.Pipeline.current()` and query for the required value::
```
Comment (jantonguirao) [inline dali/python/nvidia/dali/pipeline.py:1280]:
```suggestion
    To access any pipeline arguments within the body of a ``@pipeline_def`` function, the function
    :meth:`nvidia.dali.Pipeline.current()` can be used::
```

File: dali/python/nvidia/dali/pipeline.py [hunk 2]
```diff
@@ -1274,6 +1275,16 @@ def my_pipe():
...
+
+        @pipeline_def(batch_size=42, num_threads=3)
+        def my_pipe(flip_vertical, flip_horizontal):
+            pipe = Pipeline.current()
+            x = pipe.batch_size
+            y = pipe.num_threads
+            ...
```
Comment (jantonguirao) [inline dali/python/nvidia/dali/pipeline.py:1287]:
```suggestion
        @pipeline_def()
        def my_pipe(flip_vertical, flip_horizontal):
            pipe = Pipeline.current()
            x = pipe.batch_size
            y = pipe.num_threads
        pipe = my_pipe(batch_size=42, num_threads=3)
        ...
```

File: dali/python/nvidia/dali/pipeline.py (hunk 1 above)
Comment (JanuszL) [reply dali/python/nvidia/dali/pipeline.py]: Done

File: dali/python/nvidia/dali/pipeline.py (hunk 2 above)
Comment (JanuszL) [reply dali/python/nvidia/dali/pipeline.py]: Done

**Review by jantonguirao — APPROVED** [review]

**Review by banasraf — APPROVED** [review]

**Comment by JanuszL** [comment]
!build

**Pull request merged**